core/pom.xml

```diff
@@ -803,6 +805,28 @@ THE SOFTWARE.
           </dependency>
         </dependencies>
       </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>findbugs-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>findbugs</id>
+            <goals>
+              <goal>check</goal>
+            </goals>
+            <phase>verify</phase>
+            <configuration>
+              <effort>Max</effort>
+              <threshold>High</threshold>
+              <!--Excludes file is located on the top level-->
+              <excludeFilterFile>../src/findbugs/findbugs-excludes.xml</excludeFilterFile>
```

Comment:
did you check this works when invoked both from the top level directory as well as inside this module (`mvn clean verify` and `cd cli && mvn clean verify`?

I think this should be safer with `${project.basedir}/../src/findbugs/findbugs-excludes.xml`